src/app/api/utils/auth.tsx

```diff
@@ -26,6 +25,15 @@ import { SerializedSubscriber } from "../../../next-auth";
 import { record } from "../../functions/server/glean";
 import { renderEmail } from "../../../emails/renderEmail";
 import { SignupReportEmail } from "../../../emails/templates/signupReport/SignupReportEmail";
+import { getEnvVarsOrThrow } from "../../../envVars";
+
+const envVars = getEnvVarsOrThrow([
```

Comment:
Oh that's pretty cool, nice idea. I didn't like how non-standard `AppConstants` was but I really like the behavior of throwing upfront.